app/src/main/java/ch/epfl/polychef/fragments/FavouritesFragment.java

```diff
@@ -125,7 +141,7 @@ private boolean addNextRecipes() {
     private void setFavouriteOffline(int end, List<Recipe> favouritesList) {
         for(int i = indexFavourites; i < end; ++i) {
             dynamicRecipeList.add(favouritesList.get(i));
-            favouriteRecyclerView.getAdapter().notifyDataSetChanged();
+            Objects.requireNonNull(favouriteRecyclerView.getAdapter()).notifyDataSetChanged();
```

Comment:
That's very good, I'll use that in my code too

Reply:
Warnings taught me ^^